spec/forms/search_form_spec.rb

```diff
@@ -490,4 +490,12 @@
       expect(form.as_json).to eq(expected_hash)
     end
   end
+
+  describe 'gracefully discard unsupported params' do
+    it 'discards param pension_pot_size' do
+        expect(described_class.attribute_method? "pension_pot_size").to be_falsey
+        expect(described_class.attribute_method? "advice_method").to be_truthy
+        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
+    end
```

Comment:
This three lines has 4 spaces indentation. We're using 2 spaces on the codebase and I would recommend to be 2 spaces. Also rubocop raised some things for these lines:

```
Inspecting 83 files
.........W...............................C.........................................

Offenses:

app/forms/search_form.rb:21:30: W: Lint/UnusedBlockArgument: Unused block argument - value. If it's necessary, use _ or _value as an argument name to indicate that it won't be used.
    attributes.each do |key, value|
                             ^^^^^
spec/forms/search_form_spec.rb:496:5: C: Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.
        expect(described_class.attribute_method? "pension_pot_size").to be_falsey
    ^^^^
spec/forms/search_form_spec.rb:496:16: C: Style/NestedParenthesizedCalls: Add parentheses to nested method call described_class.attribute_method? "pension_pot_size".
        expect(described_class.attribute_method? "pension_pot_size").to be_falsey
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:496:50: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
        expect(described_class.attribute_method? "pension_pot_size").to be_falsey
                                                 ^^^^^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:497:16: C: Style/NestedParenthesizedCalls: Add parentheses to nested method call described_class.attribute_method? "advice_method".
        expect(described_class.attribute_method? "advice_method").to be_truthy
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:497:50: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
        expect(described_class.attribute_method? "advice_method").to be_truthy
                                                 ^^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:498:17: C: Layout/SpaceInsideBlockBraces: Space missing inside {.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                ^
spec/forms/search_form_spec.rb:498:37: C: Layout/SpaceInsideHashLiteralBraces: Space inside { missing.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                    ^
spec/forms/search_form_spec.rb:498:37: C: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:498:38: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                     ^^^^^^^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:498:59: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                                          ^^^^^
spec/forms/search_form_spec.rb:498:66: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                                                 ^^^^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:498:84: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                                                                   ^^^^^^^^^^^^^^
spec/forms/search_form_spec.rb:498:98: C: Layout/SpaceInsideHashLiteralBraces: Space inside } missing.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                                                                                 ^
spec/forms/search_form_spec.rb:498:100: C: Layout/SpaceInsideBlockBraces: Space missing inside }.
        expect {described_class.new({:pension_pot_size => "any", :advice_method => "face_to_face"})}.to_not raise_exception
                                                                                                   ^

83 files inspected, 15 offenses detected
```